### ISSUE TITLE
wtf /tmp/toto.edf ;)

### ISSUE THREAD

**Issue by picca** [silx-kit/pyFAI#678]
Hello guyes.

I have some images from a marccd. 2048x2048. I works greate when I open this with silx view of pyFAI-calib2 on the command line

I can confirm that

pyFAI-calib2 blabla display the right image.

BUT when I try to open a new mrccd image I get a picture 10x10 named /tmp/toto.edf

This is a pain ... ;)

**Comment by picca** [comment]
I can find this in the code

```
        self.__dialogState = dialog.saveState()
        filename = dialog.selectedFiles()[0]
        try:
            print("before opening image")
            with fabio.open(filename) as image:
                print("image is loaded")

                f = fabio.edfimage.EdfImage(data=numpy.random.random((10, 10)), header={})
                f.filename = "/tmp/toto.edf"
                yield f
                # yield image
        except Exception as e:
            _logger.error(e.args[0])
            _logger.debug("Backtrace", exc_info=True)
            # FIXME Display error dialog
            yield None
        except KeyboardInterrupt:
            raise
```

debuging purpose ?

**Comment by picca** [comment]
Ok, looking at the current code it is no more the case.

**Comment by vallsv** [comment]
Yes, that's a bug fix for the coming pyfai 0.14.2. Sorry for that.
We think about releasing it next week.
